ui/src/plugins/delete/Delete.js

```diff
@@ -60,6 +61,8 @@ export default function (config = {}) {
       createDeleteSuccessEvent()
 
       onDeleteSuccess(response)
+
+      redirectRoute && handleRedirectRoute(this)
```

Comment:
se for passado um objeto mesmo que vazio, ele vai chamar este cara.

Reply:
mas pq o cara vai passar um objeto vazio?